ingestclient/utils/filesystem.py

```diff
@@ -143,7 +146,29 @@ def get_file(self, path):
             (io.BufferedReader): A file handle for the specified file
         """
         output = six.BytesIO()
-        self.bucket.download_fileobj(path, output)
+
+        # Allows for wildcards (Involves data crawl, computationally expensive)
+        if '*' in path:
```

Comment:
There's no code path if there's no `*` in the path.

Reply:
Oops good point. Should just be downloading if there is no wildcard.